redpanda-data-console.yaml

```diff
@@ -0,0 +1,120 @@
+package:
+  name: redpanda-data-console
+  version: 2.7.2
+  description: "Redpanda Console (previously known as Kowl) is a web application that helps you manage and debug your Kafka/Redpanda workloads effortlessly."
+  epoch: 0
+  copyright:
+    - license: "BUSL-1.1"
+  resources:
+    cpu: 8
+    memory: 8Gi
```

Comment:
Why do we set resources here?

Reply:
`npm run build` for this project is a bit memory-heap hungry so i couldn't able to build with default 4Gi, but it builds with at least 8Gi.